content/_partials/downloadbanner.html.haml

```diff
@@ -30,6 +30,24 @@
         });
       }
     }
+    $( "#dropT1" ).click(function(event) {
```

Comment:
If anyone is wondering why I wrote this code again here, it's because jquery code written in downloadlist.html.haml  somehow doesn't work for popover. Maybe this is because of same origin policy thing but I am not sure..